README.md

```diff
@@ -0,0 +1,14 @@
+####  Documentation  
+
+This lab is unfinished because I have done what I can without assistance. I will find a TA for help tomorrow.
+
+## Data Structures
+  Using the src file which holds __test__ and lib. Lib holds my js and test holds my js testing. All packages and dependencies are in the tree outside of these folders but within the lab. 
+ I'm not sure what I'm to do with the following:
```

Comment:
They just want instructions and examples for how to use your code. You can look at my lab 5 readme for reference -- https://github.com/strawbee/05-linked-lists/blob/master/lab-joy/README.md